### PR TITLE
Fixed path to updated files in System Information page

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The path of API was wrong
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Check Ajax calls in System Information page (no more warning cause of 404).

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9389)

### CODE REVIEW

File: src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function getListOfUpdatedFiles($dir = null, $path = '')
         );
         
         if (is_null($dir)) {
-            $xml = @simplexml_load_file(_PS_API_URL_.'/xml/md5/'.AppKernel::VERSION.'.xml');
+            $xml = @simplexml_load_file(_PS_API_URL_.'/xml/md5-'.AppKernel::MAJOR_VERSION.'/'.AppKernel::VERSION.'.xml');
```
Comment (PierreRambaud) [inline src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php:48]: Hum... Can someone explain me why they change `xml/md5` in `xml/md5-17`? oO when we use the version in the name of the file. :cold_sweat:

Comment (mickaelandrieu) [reply src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php:48]: ping @eternoendless :)

Comment (eternoendless) [reply src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php:48]: It has been like that since the start of 1.7 as far as I know

Comment (eternoendless) [reply src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php:48]: BTW this will come back to bite us in the back if we ever get to PrestaShop v17.0 :P

Comment (PierreRambaud) [reply src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php:48]: :cry:  Is their a way to keep `xml/md5-17` and move to `xml/md5`, sure we will forget to move to `xml/md5-18` :)

**Review by eternoendless — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

File: src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php (hunk 1 above)
Comment (mickaelandrieu) [reply src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php:48]: @eternoendless I'd like to read PrestaShop 7 instead, isn't prettier? :)

As you both agree on this pr, I add the "waiting for QA" label.

**Comment by PierreRambaud** [comment]
Thanks @mickaelandrieu

**Pull request merged**